packages/docs/content/migration-guides/v9-label-updates.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Separating labels, hints, and errors in v9
+order: 60
```

Comment:
how are we ordering our migration guides? i'm looking at the list of guides we currently have and there's seemingly no system. it's not alphabetical and not chronological so i'm curious

Reply:
That's a great question...

Reply:
I'll order it by date for now